ansible/prepare_openclose_pr_tool.yml

```diff
@@ -10,7 +10,7 @@
     - name: identifier
       prompt: "Provide an unique identifier for the PR"
       private: false
-
+      
```

Comment:
This appeared because I've force pushed the first commit, in that one, was 3 additional dialogs here, and I need to delete due to review, there is no additional line here, as we can see:

```
  - name: identifier
      prompt: "Provide an unique identifier for the PR"
      private: false
      
    - name: branch
      prompt: "Branch name to open PR against it"
      private: false

    - name: repo_owner
      prompt: "Repo owner to create branches"
      private: false
      default: "freeipa-pr-ci"
```